src/components/forms/DateInput/DateInput.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { DateInput } from './DateInput'
+import { DateInputGroup } from '../DateInputGroup/DateInputGroup'
+import { Fieldset } from '../Fieldset/Fieldset'
+
+export default {
+  title: 'Form/DateInput',
+  parameters: {
+    info: `
+USWDS 2.0 DateInput component
+
+Source: https://designsystem.digital.gov/components/form-controls/#date-input
+`,
+  },
+}
+
+export const defaultDateInput = (): React.ReactElement => (
+  <DateInput
+    id="testDateInput"
+    name="testName"
+    label="Month"
+    unit="month"
+    maxLength={2}
+    minLength={2}
+  />
+)
+
+export const dateOfBirthExample = (): React.ReactElement => (
+  <Fieldset legend={'Date of birth'}>
```

Comment:
🎨 fyi can just be `legend="Date of Birth"` a `string` is a `ReactNode`. So is a number, null or several other types of values.